addons/account/views/account_account_views.xml

```diff
@@ -119,7 +119,7 @@
                         <filter string="Account Type" name="accounttype" domain="" context="{'group_by':'user_type_id'}"/>
                     </group>
                     <searchpanel class="account_root">
-                        <field name="root_id" icon="fa-filter"/>
+                        <field name="root_id" icon="fa-filter" limit="0"/>
```

Comment:
Actually, I think it is useless because of the search panel content (values are numbers with two digits max I think). But this is to emphasise we don't want any message to be displayed: the search panel must be very thin.